namespace-lock.go

```diff
@@ -91,10 +90,7 @@ func (n *nsLockMap) unlock(volume, path string, readLock bool) {
 			nsLk.Unlock()
 		}
 		if nsLk.ref == 0 {
-			log.WithFields(logrus.Fields{
-				"volume": volume,
-				"path":   path,
-			}).Error("ref count in NS lock can not be 0.")
+			errorIf(errors.New("Ref count in NS lock cannot be 0."), "Invalid reference count detected.")
```

Comment:
errorIf(errors.New("Namespace reference count cannot be 0."), "Invalid reference count detected.")